boards/riscv32/arty_bm310/doc/arty_bm310.rst

```diff
@@ -0,0 +1,65 @@
+.. _arty_bm310:
+
+CloudBEAR BM-310
+##############
+
+Overview
+********
+
+The CloudBEAR BM-310 is CloudBEAR IP design for Artix-7 35T Arty FPGA.
+More information about BM-310 can be found on
+`CloudBEAR's website <https://www.cloudbear.ru/bm_310.html>`_.
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``BM-310`` board configuration can be built as usual
```

Comment:
How about replacing these two sentences with:

    Applications for the BM-310 board can be built as usual
    (see :ref:`build_an_application`), using the ``arty_bm310`` board configuration:

Reply:
@dbkinder I agree It's better.